### ISSUE TITLE
Been getting less subtiles and a lot of error 500's (disconnected) on Bazarr lately

### ISSUE THREAD

**Issue by Tuumke** [morpheus65535/bazarr#2105]
**Describe the bug**
Can't really say since when or what update, but i'm barely getting any results on subs these days. Even though i have multiple providers enabled. A lot of times i also get "500 Disconnected from server" and in the logs its shows a: ERROR (app:1741) - ``` Exception on /api/providers/episodes [GET] ```

**To Reproduce**
Unclear. But manual search usually gives me this error

**Expected behavior**
Find subs, since a manual search on let's say addic7ed works fine and shows results for my release.

**Screenshots**

**Software (please complete the following information):**
 - Bazarr: v1.2.0-ls188 by linuxserver.io
 - Radarr version 4.4.3.7000-ls454 by [linuxserver.io](https://www.linuxserver.io/)
 - Sonarr version 3.0.10.1567-ls184 by [linuxserver.io](https://linuxserver.io/)
 - OS: docker

**Additional context**
These are the providers i enabled:
![image](https://user-images.githubusercontent.com/935617/227600533-e7fc9440-0ae7-4def-bb05-52656e2fd31b.png)

A manual search on Addic7ed for NCIS S20E17 Stranger in a Strange Land gives me 2 results, 1 of which 1 need. My anti-captcha provider also doesn't show it's being used? And the weird thing is, i use Emby Addic7ed plugin, it just works... without any captcha:

![image](https://user-images.githubusercontent.com/935617/227601101-18333407-d982-42af-9a30-c27795db5f8d.png)

**Comment by morpheus65535** [comment]
Please share your log.

**Comment by Tuumke** [comment]
[bazarr.log](https://github.com/morpheus65535/bazarr/files/11067162/bazarr.log)

**Comment by Tuumke** [comment]
[bazarr-2023-03-24.log](https://github.com/morpheus65535/bazarr/files/11067175/bazarr-2023-03-24.log)
And yesterdays log, as today seems kinda small? :S

**Comment by morpheus65535** [comment]
Largest part of your log is about connection issues to providers. Disable your proxy by clicking on the small "x" at the right end of the proxy type field. Restart Bazarr and let see if you still have that much throttled providers.

**Comment by Tuumke** [comment]
When i now manually searched, nothing happens either. Not even a "0 results" screen. Just still the search button. Logs show:

```
[custom-init] No custom files found, skipping...
2023-03-25 17:10:02,252 - root                             (14d9642e1b38) :  INFO (signalr_client:53) - BAZARR trying to connect to Sonarr SignalR feed...
2023-03-25 17:10:02,253 - root                             (14d9641deb38) :  INFO (signalr_client:190) - BAZARR trying to connect to Radarr SignalR feed...
2023-03-25 17:10:02,253 - root                             (14d9730cfb48) :  INFO (server:67) - BAZARR is started and waiting for request on http://0.0.0.0:6767
2023-03-25 17:10:02,264 - root                             (14d9640dbb38) :  INFO (signalr_client:218) - BAZARR SignalR client for Radarr is connected and waiting for events.
2023-03-25 17:10:02,335 - root                             (14d9642e1b38) :  INFO (signalr_client:73) - BAZARR SignalR client for Sonarr is connected and waiting for events.
[ls.io-init] done.
2023-03-25 17:17:17,371 - root                             (14d965985b38) :  INFO (get_providers:346) - Using addic7ed again after 10 minutes, (disabled because: AuthenticationError)
2023-03-25 17:17:17,372 - root                             (14d965985b38) :  INFO (get_providers:346) - Using podnapisi again after 10 minutes, (disabled because: SSLError)
2023-03-25 17:17:17,398 - root                             (14d965985b38) :  INFO (get_providers:346) - Using subscene again after 10 minutes, (disabled because: HTTPError)
2023-03-25 17:17:39,054 - retry.api                        (14d9623c3b38) :  WARNING (api:40) - HTTPSConnectionPool(host='www.addic7ed.com', port=443): Read timed out. (read timeout=10), retrying in 5 seconds...
2023-03-25 17:17:52,285 - root                             (14d9623c3b38) :  INFO (get_providers:299) - Throttling addic7ed for 10 minutes, until 23/03/25 17:27, because of: AuthenticationError. Exception info: 'cookies not valid anymore'
2023-03-25 17:18:34,367 - app.app                          (14d965476b38) :  ERROR (app:1741) - Exception on /api/providers/episodes [GET]
2023-03-25 17:18:36,803 - root                             (14d96a157b38) :  INFO (get_providers:378) - BAZARR throttled providers have been reset.
2023-03-25 17:19:48,390 - app.app                          (14d969b48b38) :  ERROR (app:1741) - Exception on /api/providers/episodes [GET]

```

**Comment by Tuumke** [comment]
Debugging shows that Addic7ed actually gives back results. 

```
2023-03-25 17:22:44,760 - subliminal_patch.providers.addic7ed (14a59cbb9b38) :  DEBUG (addic7ed:487) - Found subtitle <Addic7edSubtitle 'http://www.addic7ed.com/updated/1/181336/2' [en]>
2023-03-25 17:22:44,760 - subliminal_patch.providers.addic7ed (14a59cbb9b38) :  DEBUG (addic7ed:487) - Found subtitle <Addic7edSubtitle 'http://www.addic7ed.com/updated/1/181336/3' [en]>
2023-03-25 17:22:44,761 - subliminal_patch.providers.addic7ed (14a59cbb9b38) :  DEBUG (addic7ed:487) - Found subtitle <Addic7edSubtitle 'http://www.addic7ed.com/updated/10/181336/4' [pt-BR]>
2023-03-25 17:22:44,761 - subliminal_patch.providers.addic7ed (14a59cbb9b38) :  DEBUG (addic7ed:487) - Found subtitle <Addic7edSubtitle 'http://www.addic7ed.com/updated/10/181336/5' [pt-BR]>
2023-03-25 17:22:44,761 - subliminal_patch.providers.addic7ed (14a59cbb9b38) :  DEBUG (addic7ed:487) - Found subtitle <Addic7edSubtitle 'http://www.addic7ed.com/updated/1/181462/0' [en]>
2023-03-25 17:22:44,761 - subliminal_patch.providers.addic7ed (14a59cbb9b38) :  DEBUG (addic7ed:487) - Found subtitle <Addic7edSubtitle 'http://www.addic7ed.com/updated/1/181462/1' [en]>
2023-03-25 17:22:44,762 - subliminal_patch.providers.addic7ed (14a59cbb9b38) :  DEBUG (addic7ed:487) - Found subtitle <Addic7edSubtitle 'http://www.addic7ed.com/updated/1/181462/2' [en]>
2023-03-25 17:22:44,762 - subliminal_patch.providers.addic7ed (14a59cbb9b38) :  DEBUG (addic7ed:487) - Found subtitle <Addic7edSubtitle 'http://www.addic7ed.com/updated/1/181462/3' [en]>
2023-03-25 17:22:44,762 - subliminal_patch.providers.addic7ed (14a59cbb9b38) :  DEBUG (addic7ed:487) - Found subtitle <Addic7edSubtitle 'http://www.addic7ed.com/updated/10/181462/4' [pt-BR]>
2023-03-25 17:22:44,763 - subliminal_patch.providers.addic7ed (14a59cbb9b38) :  DEBUG (addic7ed:487) - Found subtitle <Addic7edSubtitle 'http://www.addic7ed.com/updated/10/181462/5' [pt-BR]>
2023-03-25 17:22:44,763 - subliminal_patch.providers.addic7ed (14a59cbb9b38) :  DEBUG (addic7ed:487) - Found subtitle <Addic7edSubtitle 'http://www.addic7ed.com/updated/1/181975/0' [en]>
2023-03-25 17:22:44,763 - subliminal_patch.providers.addic7ed (14a59cbb9b38) :  DEBUG (addic7ed:487) - Found subtitle <Addic7edSubtitle 'http://www.addic7ed.com/updated/1/181975/1' [en]>
2023-03-25 17:22:44,764 - subliminal_patch.providers.addic7ed (14a59cbb9b38) :  DEBUG (addic7ed:487) - Found subtitle <Addic7edSubtitle 'http://www.addic7ed.com/updated/1/181975/2' [en]>
2023-03-25 17:22:44,764 - subliminal_patch.providers.addic7ed (14a59cbb9b38) :  DEBUG (addic7ed:487) - Found subtitle <Addic7edSubtitle 'http://www.addic7ed.com/updated/1/181975/3' [en]>
2023-03-25 17:22:44,765 - subliminal_patch.providers.addic7ed (14a59cbb9b38) :  DEBUG (addic7ed:487) - Found subtitle <Addic7edSubtitle 'http://www.addic7ed.com/updated/10/181975/4' [pt-BR]>
2023-03-25 17:22:44,765 - subliminal_patch.providers.addic7ed (14a59cbb9b38) :  DEBUG (addic7ed:487) - Found subtitle <Addic7edSubtitle 'http://www.addic7ed.com/updated/10/181975/5' [pt-BR]>
2023-03-25 17:22:44,765 - subliminal_patch.providers.addic7ed (14a59cbb9b38) :  DEBUG (addic7ed:487) - Found subtitle <Addic7edSubtitle 'http://www.addic7ed.com/updated/1/182365/0' [en]>
2023-03-25 17:22:44,766 - subliminal_patch.providers.addic7ed (14a59cbb9b38) :  DEBUG (addic7ed:487) - Found subtitle <Addic7edSubtitle 'http://www.addic7ed.com/updated/1/182365/1' [en]>
2023-03-25 17:22:44,766 - subliminal_patch.providers.addic7ed (14a59cbb9b38) :  DEBUG (addic7ed:487) - Found subtitle <Addic7edSubtitle 'http://www.addic7ed.com/updated/1/182365/2' [en]>
2023-03-25 17:22:44,766 - subliminal_patch.providers.addic7ed (14a59cbb9b38) :  DEBUG (addic7ed:487) - Found subtitle <Addic7edSubtitle 'http://www.addic7ed.com/updated/1/182365/3' [en]>
2023-03-25 17:22:44,767 - subliminal_patch.providers.addic7ed (14a59cbb9b38) :  DEBUG (addic7ed:487) - Found subtitle <Addic7edSubtitle 'http://www.addic7ed.com/updated/10/182365/4' [pt-BR]>
2023-03-25 17:22:44,767 - subliminal_patch.providers.addic7ed (14a59cbb9b38) :  DEBUG (addic7ed:487) - Found subtitle <Addic7edSubtitle 'http://www.addic7ed.com/updated/10/182365/5' [pt-BR]>
2023-03-25 17:22:44,768 - subliminal_patch.providers.addic7ed (14a59cbb9b38) :  DEBUG (addic7ed:487) - Found subtitle <Addic7edSubtitle 'http://www.addic7ed.com/updated/1/182546/0' [en]>
2023-03-25 17:22:44,768 - subliminal_patch.providers.addic7ed (14a59cbb9b38) :  DEBUG (addic7ed:487) - Found subtitle <Addic7edSubtitle 'http://www.addic7ed.com/updated/1/182546/1' [en]>
2023-03-25 17:22:44,768 - subliminal_patch.providers.addic7ed (14a59cbb9b38) :  DEBUG (addic7ed:487) - Found subtitle <Addic7edSubtitle 'http://www.addic7ed.com/updated/1/182546/2' [en]>
2023-03-25 17:22:44,769 - subliminal_patch.providers.addic7ed (14a59cbb9b38) :  DEBUG (addic7ed:487) - Found subtitle <Addic7edSubtitle 'http://www.addic7ed.com/updated/1/182546/3' [en]>
2023-03-25 17:22:47,547 - urllib3.connectionpool           (14a59c20eb38) :  DEBUG (connectionpool:456) - https://subscene.com:443 "POST /subtitles/searchbytitle HTTP/1.1" 200 None
2023-03-25 17:22:47,566 - subliminal_patch.providers.subscene (14a59c20eb38) :  DEBUG (subscene:350) - No results found
2023-03-25 17:22:47,576 - SignalRCoreClient                (14a59e2e7b38) :  DEBUG (websocket_transport:193) - Sending message <signalrcore.messages.ping_message.PingMessage object at 0x14a5a620ce20>
```

But when i get the "Error 500 You have disconnected from the server", debug shows:

```
2023-03-25 17:23:44,798 - subliminal_patch.providers.utils (14a5a17e5b38) :  DEBUG (utils:169) - Guessit options to update matches: {'type': 'episode'}
2023-03-25 17:23:44,799 - subliminal_patch.providers.utils (14a5a17e5b38) :  DEBUG (utils:176) - Updating matches from release info: 720p-SYNCOPY
2023-03-25 17:23:44,810 - subliminal_patch.subtitle        (14a5a17e5b38) :  DEBUG (subtitle:436) - resolution matched? True (720p -> 720p)
2023-03-25 17:23:44,810 - subliminal_patch.subtitle        (14a5a17e5b38) :  DEBUG (subtitle:570) - Both video and guess don't have streaming_service. Returning True
2023-03-25 17:23:44,811 - subliminal_patch.subtitle        (14a5a17e5b38) :  DEBUG (subtitle:570) - Both video and guess don't have edition. Returning True
2023-03-25 17:23:44,811 - subliminal_patch.subtitle        (14a5a17e5b38) :  DEBUG (subtitle:570) - Both video and guess don't have other. Returning True
2023-03-25 17:23:44,811 - subliminal_patch.providers.utils (14a5a17e5b38) :  DEBUG (utils:180) - New matches: {'edition', 'streaming_service', 'year', 'resolution', 'other'}
2023-03-25 17:23:44,812 - app.app                          (14a5a17e5b38) :  ERROR (app:1741) - Exception on /api/providers/episodes [GET]
Traceback (most recent call last):
  File "/app/bazarr/bin/bazarr/../libs/flask/app.py", line 1820, in full_dispatch_request
    rv = self.dispatch_request()
  File "/app/bazarr/bin/bazarr/../libs/flask/app.py", line 1796, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/app/bazarr/bin/bazarr/../libs/flask_restx/api.py", line 405, in wrapper
    resp = resource(*args, **kwargs)
  File "/app/bazarr/bin/bazarr/../libs/flask/views.py", line 107, in view
    return current_app.ensure_sync(self.dispatch_request)(**kwargs)
  File "/app/bazarr/bin/bazarr/../libs/flask_restx/resource.py", line 46, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/app/bazarr/bin/bazarr/api/utils.py", line 30, in wrapper
    return actual_method(*args, **kwargs)
  File "/app/bazarr/bin/bazarr/../libs/flask_restx/marshalling.py", line 244, in wrapper
    resp = f(*args, **kwargs)
  File "/app/bazarr/bin/bazarr/api/providers/providers_episodes.py", line 69, in get
    data = manual_search(episodePath, profileId, providers_list, sceneName, title, 'series')
  File "/app/bazarr/bin/bazarr/subtitles/pool.py", line 85, in decorated
    return f(*args, **kwargs)
  File "/app/bazarr/bin/bazarr/subtitles/manual.py", line 86, in manual_search
    logging.debug(f"BAZARR Skipping {s}, because it doesn't match our series/episode")
TypeError: __str__ returned non-string (type bytes)
2023-03-25 17:23:47,660 - SignalRCoreClient                (14a59e2e7b38) :  DEBUG (websocket_transport:193) - Sending message <signalrcore.messages.ping_message.PingMessage object at 0x14a5a792c070>
2023-03-25 17:23:47,661 - SignalRCoreClient                (14a59e2e7b38) :  DEBUG (json_hub_protocol:49) - {"type": 6}
2023-03-25 17:23:52,667 - SignalRCoreClient                (14a59e2e7b38) :  DEBUG (websocket_transport:193) - Sending message <signalrcore.messages.ping_message.PingMessage object at 0x14a5a568d720>
2023-03-25 17:23:52,668 - SignalRCoreClient                (14a59e2e7b38) :  DEBUG (json_hub_protocol:49) - {"type": 6}
2023-03-25 17:23:53,254 - engineio.server                  (14a59cfe9b38) :  INFO (socket:89) - b53PcD4sRW1bAxOwAAAC: Sending packet PING data None
2023-03-25 17:23:53,279 - engineio.server                  (14a5a0ecab38) :  INFO (socket:52) - b53PcD4sRW1bAxOwAAAC: Received packet PONG data

```

**Comment by Tuumke** [comment]
> Fixed in development branch. Latest beta shouldn't be giving this error anymore.

Any idea what tag to use for the lsio docker?

**Comment by morpheus65535** [comment]
The fix has been added in 1.2.1-beta.8 and is already available in current `linuxeserver/bazarr:development`.

**Comment by morpheus65535** [comment]
@Tuumke no need to backup. Nice to see that it fixed your issue!
